.gitignore

```diff
@@ -1,3 +1,4 @@
 .vscode
 main
-build
\ No newline at end of file
+build
+a.out
```

Comment:
Add newline at the end of the file?

Some diff tools have a weird way of handling files that don't end with newlines, so it's always better to add a newline at the end of code files.